de4a-connector/src/main/java/eu/de4a/connector/error/utils/ErrorHandlerUtils.java

```diff
@@ -141,8 +141,12 @@ public static String postRestObjectWithCatching(String url, String request, bool
         } else {
             if(obj instanceof String) {
                 retObj = marshaller.read((String) obj);
-            } else {
+            } else if(obj instanceof InputStream) {
```

Comment:
Is that InputStream properly closed?

Reply:
hera, I think it is the caller who should be responsible for closing it, isn´t it?

Reply:
Yes okay. Maybe you can just add a comment like "Caller is expected to close"